### PR TITLE
feat(css/minifier): remove duplicate selectors

### PR DESCRIPTION
**Pull request by alexander-akait**
**Description:**

Remove duplicate selectors

**BREAKING CHANGE:**

No

**Related issue (if exists):**

No

### CODE REVIEW

File: crates/swc_css_minifier/src/compress/selector.rs [hunk 1]
```diff
@@ -15,6 +15,24 @@ struct CompressSelector {
 impl CompressSelector {}
 
 impl VisitMut for CompressSelector {
+    fn visit_mut_selector_list(&mut self, selector_list: &mut SelectorList) {
+        selector_list.visit_mut_children_with(self);
+
+        let mut already_seen: AHashSet<ComplexSelector> = Default::default();
```
Comment (kdy1) [inline crates/swc_css_minifier/src/compress/selector.rs:21]: This should be a vector instead

**Review by kdy1 — CHANGES_REQUESTED** [review]
Iterating over hashmap is slow

**Comment by alexander-akait** [comment]
@kdy1 Fixed :+1:

**Review by kdy1 — APPROVED** [review]
Thanks!

---

swc-bump:
 - swc_css_minifier

**Pull request merged**